### DISCUSSION TITLE
Introduction - Quentin

### DISCUSSION THREAD

**Discussion by TheBitcoinBackup** [MyFirstBitcoin/.github#5] [General]
Hello everyone, 

I'm Quentin, a French Bitcoiner living in El Salvador. I'm in charge of Mi Primer Bitcoin's International Expansion. 

I look forward to having enlightening discussions with like-minded Bitcoiners here on Github. Let's build together!

**Comment by DaxSosa** [comment]
Hello ...

**Comment by BabaevDaniel** [comment]
Hello everyone,

I'm Daniel, born in Israel and living in Canada. I will be translating "My First Bitcoin" from English to Russian and then Hebrew.
Please see my repositories for more information.

I'm eager to help and extend your mission worldwide!

**Comment by hectoralvero** [comment]
Thanks Quentin.  Looks good!

**Comment by HUX13Y** [comment]
Hi Quentin, thanks for your hard work in setting this up!

**Comment by BitcoinVictoriaFalls** [comment]
Hi Quentin! Watching you on video right now, haha

**Comment by Forte11Cuba** [comment]
Hello, I am Forte11. I am from Cuba. I am in the Cuba Bitcoin team
Viva Bitcoin!

**Comment by BitcoinRepublic** [comment]
I'm fascinated by the developments in El Salvador involving Bitcoin. My interest was piqued in 2021 when President Bukele distributed $30 via the Chivo Wallet. Since then, I've been delving into this revolutionary technology and feel compelled to be involved. Opportunities like this are rare, and I'm determined not to let it pass me by.

**Comment by Agalan2829** [comment]
Hola Quentin

**Comment by Agalan2829** [comment]
Hello
